src/plugins/intel_npu/tools/single-image-test/cmake/standalone.cmake

```diff
@@ -37,21 +37,28 @@ set(DEPENDENCIES
         Threads::Threads
         gflags
         openvino::runtime
-        TBB::tbb
         opencv_core
         opencv_imgproc
         opencv_imgcodecs
         npu_tools_utils
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX)
-    target_compile_options(${TARGET_NAME} PRIVATE -Wall)
+if (TBB_FOUND)
+    list(APPEND DEPENDENCIES TBB::tbb)
+else()
+    message(WARNING
+        "TBB not found. We will try to build SIT without TBB. "
+        "If OpenVINO was built with TBB, you'll likely get a linking error. "
+        "Make sure you have called setupvars or have specified TBB_DIR.")
 endif()
 
 file(GLOB SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
 
 add_executable(${TARGET_NAME} ${SOURCES})
 target_link_libraries(${TARGET_NAME} PRIVATE ${DEPENDENCIES})
+if (CMAKE_COMPILER_IS_GNUCXX)
+    target_compile_options(${TARGET_NAME} PRIVATE -Wall)
```

Comment:
it's set by OpenVINO itself https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/compile_flags/os_flags.cmake#L447

Reply:
@ilya-lavrenov thanks for the comment!
Will this option be propagated if build this tool in standalone mode?
i.e.
find_package(OpenVINO REQUIRED COMPONENTS Runtime)
link with openvino::runtime

Reply:
hm, no
only with developer package

Reply:
Okay, I've addressed this comment where applicable https://github.com/openvinotoolkit/openvino/pull/25752/commits/f370e333bfe98a47e4e71cf233fb188cbdb21b71